ui/appui-abstract/src/test/utils/IconSpecUtilities.test.ts

```diff
@@ -2,6 +2,7 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
+/* eslint-disable deprecation/deprecation */
```

Comment:
I personally think we should avoid disabling this rule across entire files because its easy for to ignore using deprecated apis

Reply:
We do this a lot -- I'll try to do a pass and weed them out of the packages, but I make no promises that I'll do it in the unit tests.

Reply:
Yea no worries, unit tests i dont care much for. And dont let me hold your pr up because of this comment, we can tackle this in a separate one